packages/api/src/services/emp-state.ts

```diff
@@ -1,95 +1,72 @@
 import * as uma from "@uma/sdk";
 import Promise from "bluebird";
 const { emp } = uma.clients;
-import { BigNumber, utils } from "ethers";
-const { parseBytes32String } = utils;
-import { asyncValues, Profile } from "../libs/utils";
+import { BatchReadWithErrors, nowS, parseBytes, Profile, toNumber, toString } from "../libs/utils";
 import { AppState, BaseConfig } from "..";
 
 type Instance = uma.clients.emp.Instance;
 type Config = BaseConfig;
 type Dependencies = Pick<
   AppState,
-  "registeredEmps" | "provider" | "emps" | "collateralAddresses" | "syntheticAddresses" | "registeredEmpsMetadata"
+  | "registeredEmps"
+  | "provider"
+  | "emps"
+  | "collateralAddresses"
+  | "syntheticAddresses"
+  | "registeredEmpsMetadata"
+  | "multicall2"
 >;
 
 export default (config: Config, appState: Dependencies) => {
-  const { registeredEmps, registeredEmpsMetadata, provider, emps, collateralAddresses, syntheticAddresses } = appState;
+  const {
+    registeredEmps,
+    registeredEmpsMetadata,
+    provider,
+    emps,
+    collateralAddresses,
+    syntheticAddresses,
+    multicall2,
+  } = appState;
   const profile = Profile(config.debug);
+  // default props we want to query on contract
+  const staticProps: [string, (x: any) => any][] = [
+    ["priceIdentifier", parseBytes],
+    ["expirationTimestamp", toNumber],
+    ["withdrawalLiveness", toNumber],
```

Comment:
@daywiss The current emp state attribute is named `withdrawLiveness`, but now the name is changed to the actual name found in the contract: `withdrawalLiveness `. How should we deal with this? Rename it to the old value or update the emp state type with the new name and inform the clients of the API about this change

Reply:
hey thanks for finding this. i would say change it to align with the contract. its a minor change that does not affect anything in use (not used in umaverse)

Reply:
if you want we can make a separate issue for this